client/components/grade-table.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Grade from './grade';
+
+const GradeTable = props => {
+  const listGrades = props.grades;
+  const gradeRows = listGrades.map(grade => {
+    return <tr key={grade.id}><Grade grade={grade.grade} name={grade.name} course={grade.course} /></tr>;
```

Comment:
It would make more sense to move the `tr` element to your `Grade` component

Reply:
Okay corrected the tr element